exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,16 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+function createGreetingForDaniel(role) {
+  if (role === danielsRole ) {
+    return `Hi, I'm ${name}, I'm a ${danielsRole}`;
+  } else {
+    return `Hi, I'm ${name}, I'm a student`;
+  }
+}
+
```

Comment:
I think you should hardcode "mentor" in your function rather than use danielsRole.